packages/lib-classifier/src/store/FieldGuideStore.spec.js

```diff
@@ -54,8 +53,16 @@ describe('Model > FieldGuideStore', function () {
 
   it('should remain in an initialized state if there is no project', function () {
     rootStore = RootStore.create({
-      fieldGuide: FieldGuideStore.create(),
-      projects: ProjectStore.create()
+      classifications: {},
+      dataVisAnnotating: {},
+      drawing: {},
+      feedback: {},
+      subjects: {},
+      subjectViewer: {},
+      tutorials: {},
+      workflows: {},
+      workflowSteps: {},
+      userProjectPreferences: {}
     }, { client: { panoptes: { get: sinon.stub().callsFake(() => Promise.resolve(null)) } } })
```

Comment:
These lines get repeated throughout these tests. Could the root store be returned by a function, `rootStore = getRootStore(options)`, where options is an object that can be merged into the default snapshot before it's passed to `create()`? Then you'd only be declaring the default values once.